src/tess_asteroids/__init__.py

```diff
@@ -1,10 +1,17 @@
 import logging
+from os import path
+
+import pandas as pd
 
 # Configure logging
 logger = logging.getLogger(__name__)
 logger.addHandler(logging.StreamHandler())
 
-from .tpf import MovingTargetTPF  # noqa: E402
+# Read in straps table
+loc = path.abspath(path.dirname(__file__))
+straps = pd.read_csv(f"{loc}/data/straps.csv", comment="#")
```

Comment:
I included it in the __init__ file so that if someone creates TPFs for multiple objects, the strap table doesn't need to be loaded in multiple times. I would prefer to leave it there.

Reply:
Ok, I'm convinced. It's fine to keep it there.